### PR TITLE
Convert ValueError to ImportError

### PR DESCRIPTION
**Pull request by nd-02110114**
## What I did

Instead of ValueError, I modified that classes throws `ImportError` when the soft dependencies are not installed. `ImportError` is more appropriate about soft dependencies's problem.

### CODE REVIEW

File: deepchem/models/tests/test_reload.py [hunk 1]
```diff
@@ -288,8 +288,6 @@ def test_normalizing_flow_model_reload():
   import tensorflow_probability as tfp
   tfd = tfp.distributions
   tfb = tfp.bijectors
-  tfk = tf.keras
-  tfk.backend.set_floatx('float64')
```
Comment (nd-02110114) [inline deepchem/models/tests/test_reload.py:292]: In reload test of the normalizing flows, these lines sets all float values as `float64 ` in all tests after this test. I think these leads to failure of many tests.

**Comment by nd-02110114** [comment]
And, I decided to comment out the codes leads to many CI errors. Currently, the CI in master branch failed many tests. This is because the CI failure in master branch blocked the nightly build release and we tended to miss the original error of each PR. 

See https://pypi.org/project/deepchem/#history. The release has stopped since 10/22.

**Comment by rbharath** [comment]
@nd-02110114 Awesome, thank you for debugging the Travis CI! I was unfortunately not able to keep track of this with my neck injury. I'm going to go ahed and merge this PR in so we can get Travis back online and we can fix the commented out tests in subsequent PRs
